### PR TITLE
Close AGENDA#342: link to consent form when emailing patient.

### PR DESCRIPTION
**Pull request by dhMuse**
Request was for mock-up, but the work here should satisfy requirements. Nonetheless, perhaps best to test separately from other topic branches, or else merge in last for easy disentanglement.

### CODE REVIEW

**Comment by bgracie** [comment]
yes, let's test it separately please...

File: app/views/mail_to_patient_mailer/mail_appointment_info.html.haml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+!!!
+%html
+  %head
+    %title #{@user.name} has sent you instructions for your appiontment
+
+  %body{ style: "background-color: #f0ede8; font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;" }
+    .content-container{ style: "margin: 40px; margin-right: 10%; margin-left: 10%; padding: 20px; background-color: white" }
+
+      %p
+        #{@user.name} has sent you instructions for your appiontment with #{@profile.label}.
```
Comment (bgracie) [inline app/views/mail_to_patient_mailer/mail_appointment_info.html.haml]: speling!

Comment (dhMuse) [inline app/views/mail_to_patient_mailer/mail_appointment_info.html.haml]: :P

**Comment by dhMuse** [comment]
Closed pending approval of User Group.

**Pull request closed without merging**